src/seedu/addressbook/ui/TextUi.java

```diff
@@ -165,4 +166,11 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    public String getPrintableString(Printable... printables){
+        String returnValue = printables[0].getPrintableString();
+        for (int i=1; i<printables.length; i++){
```

Comment:
You can also take a look at Java StringBuilder, StringJoiner, and String.Join options.
You can also substitute the for loop with for each loop. Syntax : for (Printable p : printables) { }